demo/api/src/app.module.ts

```diff
@@ -110,45 +104,32 @@ export class AppModule {
                     reservedPaths: ["/events"],
                 }),
                 RedirectsModule.register({ customTargets: { news: NewsLinkBlock }, Scope: RedirectScope }),
-                BlobStorageModule.registerAsync({
-                    imports: [ConfigModule],
-                    useFactory: async () => ({
-                        blobStorageConfig: {
-                            backend: {
-                                driver: config.blob.storageDriver,
-                                file: config.blob.storageDriver === "file" ? config.fileStorage : undefined,
-                                azure: config.blob.storageDriver === "azure" ? config.azure : undefined,
-                                s3: config.blob.storageDriver === "s3" ? config.s3 : undefined,
-                            } as BlobStorageConfig["backend"],
-                        },
-                    }),
+                BlobStorageModule.register({
+                    backend: {
```

Comment:
Should we remove the `backend` "level" and move the config to the root, as it's the only key?

Reply:
hm, I'm not sure - then we have "file" at root level (of the module config)

Reply:
Okay, let's leave it like this. Maybe someday there may be additional options to the module that are not directly connected to the used backend.